### ISSUE TITLE
timeout unit is inconsistent

### ISSUE THREAD

**Issue by blampe** [uber/tchannel-python#207]
> timeout – The timeout (in seconds) for the initial advertise attempt. Defaults to 30 seconds.

> timeout (int) – How long to wait (in ms) before raising a TimeoutError - this defaults to tchannel.glossary.DEFAULT_TIMEOUT.

```
future = tchannel.thrift(
    thrift_service.getItem('foo'),
    timeout=1,  #  1 second
)
```

Our docs are inconsistent and wrong. We should be using ms everywhere. Let's fix advertise to use milliseconds

cc @abhinav @junchaowu @breerly @tangyan

**Comment by junchaowu** [comment]
why to use ms? In python, default time unit is second.

**Comment by junchaowu** [comment]
It's true we need to update the docstring.

**Comment by blampe** [comment]
Python doesn't have a default time unit. IMO ms forces you to set aggressive timeouts. A timeout of 30000 looks ridiculous and should make you feel dirty :)

**Comment by blampe** [comment]
Interesting, after looking at `concurrent.futures` the [result](https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.Future.result) method also uses a float for seconds.

A common use case in the sync case would look like this then:

```
future = tchannel.thrift(..., timeout=1000)
result = future.result(timeout=1.0)
```

Now I'm thinking maybe we should just go with floats-seconds for consistency. We could provide backwards compat (ish) by dividing any timeouts over 60 or something.

Thoughts?

**Comment by breerly** [comment]
Fixed in 0.18.0 - we're now using the more Pythonic seconds everywhere
